frontend/components/friends/profile-block.tsx

```
@@ -13,7 +13,7 @@ import {
   DialogTrigger,
 } from "../ui/dialog";
 import { Button } from "../ui/button";
-import { Github, Linkedin, User2Icon, UserRound } from "lucide-react";
+import { Github, Linkedin, User2Icon, UserRound, ExternalLink } from "lucide-react";
```

ESLint

[error] 'UserRound' is defined but never used.

(@typescript-eslint/no-unused-vars)